### PR TITLE
feat(PT-5299): Create fork of summernote to enable us to create fixes and releases as needed

### PR DESCRIPTION
**Pull request by gbtec-philippmueller**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -75,14 +75,47 @@ jobs:
...
-        if: steps.actual-branch.outputs.actual_branch == 'main'
         uses: actions/setup-node@v3
         with:
           node-version: ${{ env.NODE_VERSION }}
-          registry-url: ${{ secrets.JFROG_NPM_PUBLISH_URL }}
+          cache: npm
+          registry-url: ${{ secrets.REGISTRY_PUBLISH_URL }}
+
+      - name: Get package info
+        if: steps.actual-branch.outputs.actual_branch != 'main'
```
Comment (gbtec-stefaniesiebert) [inline .github/workflows/build.yml:85]: It's main and not master, yes?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -75,14 +75,47 @@ jobs:
...
+        uses: codex-team/action-nodejs-package-info@v1
+        id: package-info
+
+      - name: Sanitize branch name
+        uses: mad9000/actions-find-and-replace-string@2
+        if: steps.actual-branch.outputs.actual_branch != 'main' && steps.actual-branch.outputs.actual_branch != 'develop'
```
Comment (gbtec-stefaniesiebert) [inline .github/workflows/build.yml:91]: .. develop? not `development`? Why breaking our naming convention here?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -75,14 +75,47 @@ jobs:
...
+        uses: reedyuk/npm-version@1.1.1
+        with:
+          version: ${{ steps.package-info.outputs.version}}-${{steps.sanitized-branch-name.outputs.value}}
+
+      - name: Remove existing feature snapshot package
+        if: steps.actual-branch.outputs.actual_branch != 'main' && steps.actual-branch.outputs.actual_branch != 'develop'
+        continue-on-error: true
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.REGISTRY_TOKEN }}
+        run: npm unpublish --force ${{ steps.package-info.outputs.name }}@${{ steps.package-info.outputs.version}}-${{steps.sanitized-branch-name.outputs.value}}
```
Comment (gbtec-stefaniesiebert) [inline .github/workflows/build.yml:109]: do we need to force it?

**Review by gbtec-stefaniesiebert — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (gbtec-philippmueller) [reply .github/workflows/build.yml:85]: Yes, it's a fork, so we take over their branches too. `main` is the new default name for new Github repositories.

File: .github/workflows/build.yml (hunk 2 above)
Comment (gbtec-philippmueller) [reply .github/workflows/build.yml:91]: Same here, it's the branch name taken over automatically from the fork's remote.

File: .github/workflows/build.yml (hunk 3 above)
Comment (gbtec-philippmueller) [reply .github/workflows/build.yml:109]: Unfortunately yes, npm has a "security" mechanism that seems to block publishing/unpublishing within a certain range of hours if it happens to often. We need to bypass it for our CI here.

**Pull request merged**